core/include/traccc/edm/measurement.hpp

```diff
@@ -39,6 +39,9 @@ struct measurement {
     /// Geometry ID
     detray::geometry::barcode surface_link;
 
+    // Unique measurement ID
+    std::size_t measurement_id = 0;
```

Comment:
I'm a tiny bit afraid of this new variable, but if we need it, we need it...

Reply:
@stephenswat said on the `traccc` Mattermost on January 25 at 11:55am:
> Adding some additional fields to these structures should be no problem at all :slightly_smiling_face:

I need this variable to make the greedy solver work (it is needed to find all the common hits between tracks), although I was a bit reluctant to change the code, at first.